charts/ingressmonitorcontroller/values.yaml

```diff
@@ -78,3 +78,9 @@ service:
 env: []
 
 envFrom: []
+
+kube-rbac-proxy:
+  image:
+    repository: quay.io/brancz/kube-rbac-proxy
```

Comment:
Why are we using `quay.io/brancz/kube-rbac-proxy` instead of `gcr.io/kubebuilder/kube-rbac-proxy`? Can you provide a solid reason for that since `gcr.io/kubebuilder/kube-rbac-proxy` works perfectly fine for kubernetes.